### PR TITLE
Cleaning description of the teams

### LINKED ISSUE #850: Tidy up some teams
We need some help cleaning up the descriptions for some of our teams.

For all the `*-admins` teams, we need to update the descriptions to be something like:
```
admins access to the <repo> repo
```

For all the `*-maintainers` teams, we need to update the descriptions to be something like:
```
write access to the <repo> repo
```

And we can remove all the `*-reviewers` teams.

All these configs should be under the `config/` directory. They might be in any of the delegated sig yaml files.

---

If you'd like to do this, please comment that you'd like to on this issue. If you have questions, please ask them here or message me on Slack.

When you open a PR, please `/assign @cblecker`.

### PR DESCRIPTION
**Pull request by nzoueidi**
Fixes #850 

/assign @cblecker

### CODE REVIEW

File: config/kubernetes-csi/org.yaml [hunk 1]
```diff
@@ -220,126 +220,126 @@ teams:
...
   external-snapshotter-maintainers:
-    description: write access to external-snapshotter
+    description: Write access to external-snapshotter repo
     members:
     - childsb
     - saad-ali
     privacy: closed
   kubernetes-csi-github-io-admins:
-    description: admin access to kubernetes-csi.github.io
+    description: Admin access to kubernetes-csi.github.io
```
Comment (cblecker) [inline config/kubernetes-csi/org.yaml]: Missing ` repo` at end.

File: config/kubernetes/org.yaml [hunk 2]
```diff
@@ -1441,47 +1388,42 @@ teams:
...
+    description: Admin access to the kube-state-metrics repo
     members:
     - fabxc
     - ghodss
     privacy: closed
   kube-state-metrics-maintainers:
-    description: ""
+    description: Write access to the kube-state-metrics repo
     privacy: closed
   kube-ui-maintainers:
```
Comment (cblecker) [inline config/kubernetes/org.yaml]: Let's just get rid of this team completely if the repo is archived.

**Comment by justaugustus** [comment]
@nzoueidi -- can you rebase this?

**Comment by nikhita** [comment]
We can also get ride of these teams. The second team doesn't even have any members. :upside_down_face: 

https://github.com/kubernetes/org/blob/a296b35f4ac1f48fa1e8fad36f8f5c0ee8c38381/config/kubernetes-incubator/org.yaml#L381-L388

**Comment by nzoueidi** [comment]
> @nzoueidi -- can you rebase this?
 Yep, I will do it right now. I am trying to resolve the merge conflict I did :').
@nikhita should we remove both teams or we keep the first one? If we want to keep the first team we need also to rename it to `admins-incubator`

**Comment by nikhita** [comment]
> @nikhita should we remove both teams or we keep the first one?

I think we can remove both teams because they don't really have any purpose. Only the github admin team has admin access to the incubator org so `incubator-admin` isn't really useful. :)

File: config/kubernetes/org.yaml [hunk 3]
```diff
@@ -1574,7 +1480,7 @@ teams:
     - timothysc
     privacy: closed
   kubernetes-blog-maintainers:
-    description: Maintaining the Kubernetes blog on kubernetes.io
+    description: Write access on kubernetes.io
```
Comment (rlenferink) [inline config/kubernetes/org.yaml]: I think this doesn't describe what the team is actually responsible for.
`kubernetes-blog-maintainers` is assigned read access to k/website and is represented in [the blog OWNERS file](https://github.com/kubernetes/website/blob/master/content/en/blog/OWNERS). 

I don't know about `kubernetes-blog`, but it looks like that one is mentioned as team [here](https://github.com/kubernetes/community/blob/b6dcdba633c2709cfd86fe39844e360163eed6f8/sigs.yaml#L1152).

Maybe it is best to keep the description for now, or describe it as `Maintainers of the Kubernetes blog` and then when @zacharysarah is back from his holiday discuss whether we can consolidate the teams.

Comment (nzoueidi) [reply config/kubernetes/org.yaml]: That make sense, I will update it.

**Comment by cblecker** [comment]
/lgtm
/approve

@nzoueidi Thank you so much for this! This was really helpful cleanup. 🚀

**Pull request merged**